### ISSUE TITLE
[Component Request] Mind Map

### ISSUE THREAD

**Issue by IcToxi** [thedirtyfew/dash-extensions#161]
Hi there,

Are you guys interested in making a component similar to [xmind](https://www.xmind.net/)?

I want to pass in JSON to build a mind map, with callbacks to add and remove nodes.

BR

**Comment by IcToxi** [comment]
https://github.com/kozeghong/simple-mindmap
This looks good, but the author seems to have not been in GitHub for a long time.

**Comment by emilhe** [comment]
Yeah, I would prefer a more active package. Did you try `Mermaid`,  `Lottie` or [draw.io](https://app.diagrams.net)?

**Comment by IcToxi** [comment]
These are great components and I would go for them first when needed. And draw.io is a bit complicated, and I think as a dash component should be kept simple and specific for ease of construction.

If you are interested in the weather component, you can also take a look at [this](https://github.com/IcToxi/dash_weather), wrapped from [here](https://github.com/farahat80/react-open-weather).

These are also not on my immediate needs list, just that I think I will use them someday.

I'm new to React, and it may take me quite a while to figure out how you guys develop these components. I probably won't be able to contribute much until then. Thank you, I am counting on you. 😋
